<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>649</width>
    <height>542</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>631</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>设备设置</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>设备</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cBoxDevice">
     <property name="geometry">
      <rect>
       <x>58</x>
       <y>20</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>USBCAN-2E-U</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>USBCAN1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>USBCAN2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>USBCAN-E-U</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>PCIECANFD-100U</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>PCIECANFD-200U</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>PCIECANFD-400U</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>USBCANFD-100U</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>USBCANFD-MINI</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>20</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>索引号</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cBoxSuoYin">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>20</y>
       <width>69</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>0</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>6</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="cBoxChannel">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>20</y>
       <width>69</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>20</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>通道</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cBoxMode">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>正常</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>只听</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="cBoxBoud">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>50</y>
       <width>69</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>1Mbps</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>800kbps</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>500kbps</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>250kbps</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>125kbps</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>100kbps</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>50kbps</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>20kbps</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>10kbps</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>5kbps</string>
      </property>
     </item>
    </widget>
    <widget class="QCheckBox" name="checkBoxBound">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>50</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>自定义波特率</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lEditBound">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>50</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>模式</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>50</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>波特率</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_2">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>20</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>终端电阻使能</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>滤波模式</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>80</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>验收码(0x)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>80</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>屏蔽码(0x)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lEditYanShouMa">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>80</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>00000000</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lEditPingBiMa">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>80</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>FFFFFFFF</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cBoxLvbo">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>80</y>
       <width>69</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>单滤波</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>双滤波</string>
      </property>
     </item>
    </widget>
    <widget class="QCheckBox" name="cBoxAllChannel">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>20</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>所有通道</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnOpenDevice">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>打开设备</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnInitDevice">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>140</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>初始化CAN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStartDevice">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>140</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>启动CAN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnResetDevice">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>140</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>复位</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCloseDevice">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>140</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>关闭设备</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>180</y>
      <width>631</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>设备设置</string>
    </property>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>ID(0x)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>20</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>帧类型</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>20</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>协议</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>数据(0x,以空格隔开)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>50</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>发送方式</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>20</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>CANFD加速</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lEdit_ID">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>00000001</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cBoxZhenType">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>20</y>
       <width>69</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>扩展帧</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>标准帧</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="cBoxProtocol">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>20</y>
       <width>69</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>CAN</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>CANFD</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="cBoxJiaSu">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>20</y>
       <width>69</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>否</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>是</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="lEditSendData">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>50</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>00 11 22 33 44 55 66 77</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cBoxSendType">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>50</y>
       <width>69</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>正常发送</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>单次发送</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>自发自收</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="btnSend">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>70</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>发送</string>
     </property>
    </widget>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>280</y>
      <width>431</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClear">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>380</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>清空</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDeviceMessage">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>140</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>设备信息</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>649</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
